testinfra/utils/ansible_runner.py

```diff
@@ -18,150 +18,172 @@
 from __future__ import unicode_literals
 from __future__ import absolute_import
 
-import pprint
+import json
+import os
+import shutil
+import subprocess
+import tempfile
+import yaml
 
 
-try:
-    import ansible
-except ImportError:
-    raise RuntimeError(
-        "You must install ansible package to use the ansible backend")
+# the real ansible-runner
+import ansible_runner
 
-import ansible.cli.playbook
-import ansible.constants
-import ansible.executor.task_queue_manager
-import ansible.inventory
-import ansible.parsing.dataloader
-import ansible.playbook.play
-import ansible.plugins.callback
-import ansible.utils.vars
-import ansible.vars
 
-try:
-    from ansible.module_utils._text import to_bytes
-except ImportError:
-    from ansible.utils.unicode import to_bytes
+__all__ = ['AnsibleRunner', 'to_bytes']
 
 
-__all__ = ['AnsibleRunner', 'to_bytes']
+def to_bytes(data):
+    '''Why!?!?'''
+    return b'%s' % data
```

Comment:
This will most likely fail when trying to put Unicode into bytes.

cc @abadger

Reply:
<nod>. I realize that the current version of the code has removed this function but just wanted to confirm what webknjaz said.  

Any to_bytes style method has to be built around the encode method for the conversion with additional conditional logic around it to protect from trying to convert the wrong types and to set the correct encoding.